packages/@aws-cdk/aws-cognito/lib/user-pool.ts

```diff
@@ -34,6 +34,14 @@ export interface SignInAliases {
    * @default false
    */
   readonly preferredUsername?: boolean;
+
+  /**
+   * Whether sign-in aliases should be evaulated with case sensitivity.
+   * For example, when this option is set to False, users will be able to sign in using either "username" or "Username".
+   * This option enables username, preferredUsername, and email aliases to be case insensitive.
```

Comment:
generalize here to maybe just say sign-in aliases (if that's what a user should expect). is this likely to go out of date If a new sign-in alias type is added?

Reply:
Good call. I'll just remove this third line, I think the first line sums it up pretty well.